server/routes.ts

```diff
@@ -169,6 +168,40 @@ class Routes {
   /**
    * GROUPS
    */
+  ////////////////////////////////////////
+  @Router.get("/allgroups")
```

Comment:
knit: this route can just be /groups since its the only GET request with that path

Reply:
I utilized /allgroups because you've already utilized /group path for createGroup in the routes.ts